### DISCUSSION TITLE
Suggestion: Shortcut for fullscreen mode

### DISCUSSION THREAD

**Discussion by fuadalasgarov** [SpotX-Official/SpotX#458] [Ideas]
I have been using this app for months now and know what I suggest isn't even on regular Spotify. I think that having a shortcut (button F, for example) for full-screen mode would be a nice addition to make it comfortable to use the app. Thank you for reading and I apologize if this is a repeat suggestion.

**Comment by amd64fox** [comment]
I don't remember exactly, but it seemed to be in the native client, check [here](https://support.spotify.com/md-en/article/keyboard-shortcuts)
